### ISSUE TITLE
[Bug]: Rapid Fire

### ISSUE THREAD

**Issue by Deadlineem** [Yimura/YimMenu#3355]
### Describe the bug

Rapid fire only works on throwables and Weapons in the same category as the RPG (Grenade launcher, snowball launcher, etc.) and does not work for any of the other weapons.

### Steps To Reproduce

Enable rapid fire

Try shooting any weapon (like AR, Pistol, MG, Sniper)

### Expected Behavior

Rapid Fire for all.

### Logs

N/A

### Store

Epic Games

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by gir489returns** [comment]
Are you sure? It's probably just not doing anything visual, try aiming at something like a car and shooting and see if it explodes. Also, there's a note saying it can't work on shotguns because it'll fill the pool and crash.

**Comment by Deadlineem** [comment]
> Are you sure? It's probably just not doing anything visual, try aiming at something like a car and shooting and see if it explodes. Also, there's a note saying it can't work on shotguns because it'll fill the pool and crash.

Yessir, it used to be more of a visual too, but now even when shooting the gun its just one bullet at a time.  Also, network.send_chat_message and the player chat message seem broken, they are not showing up for the other players in the lobby.
![Screenshot_81](https://github.com/user-attachments/assets/06877ba6-231b-4493-97dc-f2437cac6a14)
I asked if they saw this and sent it via the button in my lua script, the reply was that.

**Comment by gir489returns** [comment]
There's nothing in the code that would stop it from working, it's entirely native based, so there's nothing to update.

**Comment by CypherMorgan** [comment]
> Yessir, it used to be more of a visual too, but now even when shooting the gun its just one bullet at a time. Also, network.send_chat_message and the player chat message seem broken, they are not showing up for the other players in the lobby. 

Afaik, if you're using a combination of words which R* might Censor, they wont appear in the chat. ( while using the menu, other than that, w/o the menu being loaded, u can use the chat and it will normally get censored ). 
And.. in this update the words like "jumped out" are also getting censored for some weird reasons, it will appear as "jum****ut* or something like that.

**Comment by gir489returns** [comment]
>And.. in this update the words like "jumped out" are also getting censored for some weird reasons, it will appear as "jum***_ut_ or something like that.

[ROCKSTAR IS HIDING THE TRUTH!](https://youtu.be/Hc9fVUoGmQo?t=18)

**Comment by Deadlineem** [comment]
> > Yessir, it used to be more of a visual too, but now even when shooting the gun its just one bullet at a time. Also, network.send_chat_message and the player chat message seem broken, they are not showing up for the other players in the lobby.
> 
> Afaik, if you're using a combination of words which R* might Censor, they wont appear in the chat. ( while using the menu, other than that, w/o the menu being loaded, u can use the chat and it will normally get censored ). And.. in this update the words like "jumped out" are also getting censored for some weird reasons, it will appear as "jum***_ut_ or something like that.

Thats definitely not the issue, my addon has a custom chat feature to send chat messages (because sometimes it breaks and i cant press t to type so i use that)

I sent "can you see this message?" And nothing went through, did this about 6 times.

**Comment by CypherMorgan** [comment]
> I sent "can you see this message?" And nothing went through, did this about 6 times.

I tried sending this, it worked normally. And when i tried saying R* is weird, it didnt went through. (Used ur addon as well as the normal and Menu's chat option too)

**Comment by gir489returns** [comment]
@Deadlineem It's possible Rockstar has flagged your account as a spammer. They are currently dealing with an ad-campaign by cheat sellers that just session hops, spams an advertisement for their cheat, and then leaves. If your account has been reported/flagged, this could be why it's not working.

**Comment by Deadlineem** [comment]
> @Deadlineem It's possible Rockstar has flagged your account as a spammer. They are currently dealing with an ad-campaign by cheat sellers that just session hops, spams an advertisement for their cheat, and then leaves. If your account has been reported/flagged, this could be why it's not working.

Its possible, but the account is about a month old max since I recently caught a 1 month ban so probably not as likely as it would have been on my main

**Comment by maybegreat48** [comment]
>  @Deadlineem It's possible Rockstar has flagged your account as a spammer. They are currently dealing with an ad-campaign by cheat sellers that just session hops, spams an advertisement for their cheat, and then leaves. If your account has been reported/flagged, this could be why it's not working.

Nah, it's just that the send message function broke after the update

**Comment by gir489returns** [comment]
> Nah, it's just that the send message function broke after the update

Do you know how to fix it?

**Comment by Yimura** [comment]
What's this issue about currently? About rapid fire not working or about the chat not working?

Please clarify so we can accurately modify the title of what's required to fix it.

**Comment by Deadlineem** [comment]
> What's this issue about currently? About rapid fire not working or about the chat not working?
> 
> Please clarify so we can accurately modify the title of what's required to fix it.

The main issue was the rapid fire, i tossed the chat one in as well bc i found out after writing this that it was broken

**Comment by Yimura** [comment]
Yeah but currently I have no clue on what is the actual state of this issue, has anyone confirmed this or is some other feature preventing this from executing normally?
